DigiMeSDK/Core/Classes/Errors/NSError+Auth.h

```diff
@@ -17,12 +17,14 @@ typedef NS_ENUM(NSInteger, AuthError) {
     AuthErrorCancelled  = 5, //authorization cancelled
     AuthErrorInvalidSession = 7, //invalid session
     AuthErrorInvalidSessionKey = 10, //session key returned by digi.me app is invalid
+    AuthErrorFailedToRetriveContract = 11,
```

Comment:
Again, this is quite a specifically named enum. The error may not **actually** have anything to do with contract fetch, right?

Looking at the available enums, it looks like we already have suitable candidate we could use in this case - `AuthErrorGeneral`